AGENTS.md

```
@@ -1,58 +1,90 @@
+ PrairieLearn is an educational learning platform with a focus on automated assessments.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)